redner.h

```diff
@@ -23,6 +23,17 @@
 #define M_PI 3.14159265358979323846
 #endif
 
+#if defined(__GNUC__) || defined(__clang__) || defined(__xlc__)
+#define USE_GCC_INTRINSICS
+#elif defined(_MSC_VER)
+#define USE_MSVC_INTRINSICS
+#include <intrin.h>
+// Define some MSVC replacements for the GCC intrinsics
+#define msvc_popcount(x) __popcnt(x)
```

Comment:
Why not use the gcc name, so that it is not necessary to use `#if` below?

Reply:
I initially did but I choose the safe side as double underscore identifiers are always reserved to implementation according to the C++ standard and defining those yourself can potentially result in undefined behavior.